client/me/account/main.jsx

```diff
@@ -759,7 +759,7 @@ const Account = createReactClass( {
 						type="button"
 						onClick={ this.getClickHandler( 'Change Username Button', this.submitUsernameForm ) }
 					>
-						{ translate( 'Save Username' ) }
+						{ translate( 'Save username' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.